### ISSUE TITLE
Consider providing type definitions for downstream users

### ISSUE THREAD

**Issue by jvican** [tnajdek/zotero-api-client#28]
There's currently no type definitions published for this library. Types not only come in handy but are sometime needed when wanting to reuse js/ts code from other programming languages. Any chance we could publish them, maybe with the help of DefinitelyTyped?

**Comment by tnajdek** [comment]
I agree, it would be a nice, I'll try to make this happen but unfortunately won't be able any time soon.
